### DISCUSSION TITLE
Bug: There is no way to play a video with Playback Restrictions

### DISCUSSION THREAD

**Discussion by danzool** [muxinc/elements#398] [Q&A] [answered]
### Which Mux Elements/Packages has a bug?

- mux-video

### Which browser(s) are you using?

- Safari
- Firefox

### Which operating system(s) are you using?

macOS

### How are you using Mux Elements?

npm package

### Describe the bug

There does not appear to be a way to instantiate a MuxPlayer and have it play a signed URL (using playback restrictions). I am signing the URL on the server side for security purposes, and so I cannot merely provide a playback ID and have the video play.

**Comment by gkatsev** [comment, marked as answer]
Hi, thanks for your patience.
It should be possible to use signed urls with `<mux-video>`. Right now, you'll want to add the `?token=` query parameter to the end of the playback id provided to `<mux-video>`
like so:
```
playback-id="DS00Spx1CV902MCtPj5WknGlR102V5HFkDe?token=jwt-signed-token"
```
It is mentioned in the mux-video [README](https://github.com/muxinc/elements/tree/main/packages/mux-video#advanced-signed-urls-and-other-playback-query-params) but I'm sure it can be easily overlooked.

Hope this helps!
